app/src/main/java/com.sun.training.ut/ui/exercise_one/ExerciseOneActivity.kt

```diff
@@ -16,5 +17,23 @@ class ExerciseOneActivity : BaseActivity<ActivityExerciseOneBinding, ExerciseOne
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         viewBinding.lifecycleOwner = this
+        viewBinding.apply {
+            pickerTimeOrder.setIs24HourView(true)
+            btnCalculate.setOnClickListener {
+                if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
+                    viewModel?.calculatorTotalAmount(
+                        hour = pickerTimeOrder.hour,
+                        voucher = switchIsHaveVoucher.isChecked,
+                        beersAmount = inputBearsNumber.text.toString().toIntOrNull() ?: 0
+                    )
+                } else {
+                    viewModel?.calculatorTotalAmount(
+                        hour = pickerTimeOrder.currentHour,
+                        voucher = switchIsHaveVoucher.isChecked,
+                        beersAmount = inputBearsNumber.text.toString().toIntOrNull() ?: 0
+                    )
+                }
```

Comment:
Cái này mình có thể viết tốt hơn là tính toán hour bên ngoài dựa vào điều kiện Build.VERSION.SDK_INT
Chỉ cần gọi viewModel?.calculatorTotalAmount 1 lân thôi, truyền các tham số đã xử lý vào, tránh tình trạng gọi lại nhiều lần, hàm mà có nhiều tham số, sau này có problem phải sửa lại nhiều chỗ.